src/harvest_translator.py

```diff
@@ -1,54 +1,68 @@
-        'region_name',
-        'elevation',
-        'elevation_unit',
-        'value',
-        'cycletime',
-        'forecast_hour',
-        'ensemble_member'
-    ],
-)
-
-"""
-Expected output from inc_logs harvester 
-inc_logs_harvested_data = namedtuple(
-    'HarvestedData',
-    [
-        'logfile',
-        'cycletime',
-        'statistic',
-        'variable',
-        'value',
-        'units'
-    ]
-)
+"""MetricTableData defines the data structure for what is stored in the 
+database, which corresponds to the db columns.
 """
+
+MetricTableData = namedtuple('MetricData', ['name',
+                                            'region_name',
+                                            'elevation',
+                                            'elevation_unit',
+                                            'value',
+                                            'cycletime',
+                                            'forecast_hour', 
+                                            'ensemble_member'])
+
 def inc_logs_translator(harvested_data):
+    """Expected output from inc_logs harvester:
+    
+    inc_logs_harvested_data = namedtuple('HarvestedData', ['logfile',
+                                                           'cycletime',
+                                                           'statistic',
+                                                           'variable',
+                                                           'value',
+                                                           'units'])
+    """
+    
     result = MetricTableData(
-        harvested_data.statistic + "_" + harvested_data.variable,
-        'global',
-        None,
-        'N/A',
-        harvested_data.value, 
-        harvested_data.cycletime,
-        None, 
-        None
-    )
+                       harvested_data.statistic + "_" + harvested_data.variable,
+                       'global',
+                       None,
+                       'N/A',
+                       harvested_data.value, 
+                       harvested_data.cycletime,
+                       None, 
+                       None)
+    return result
 
+
+def precip_translator(harvested_data):
+    """Expected output from precip harvester:
+        
+    precip_harvested_data = namedtuple('HarvestedData', ['filenames',
+                                                         'statistic',
+                                                         'variable',
+                                                         'value',
+                                                         'units',
+                                                         'mediantime',
+                                                         'longname'])
+    """
+    
+    result = MetricTableData(
+                       harvested_data.statistic + "_" + harvested_data.variable,
+                       'global',
+                       None,
+                       'N/A'
```

Comment:
You missed a comma after 'N/A' that will blow this up when running

Reply:
nice catch - added and updated the PR